dhall-try/index.html

```diff
@@ -222,18 +225,36 @@ <h1 class="text-center">The Dhall configuration language</h1>
       </div>
     </div>
     <div class="container jumbotron" style="background-color: #c5eccf">
-      <h1 class="display-4">Human-friendly</h1>
-      <p class="lead">Dhall supports comments, multi-line string literals and
-      string interpolation with non-technical users in mind.</p>
-      <textarea id="highlight0"></textarea>
-      <p>You can also automatically remove all indirection in any Dhall code,
-      converting the file to a logic-free normal form for non-programmers to
-      understand.</p>
-      <a href="https://github.com/dhall-lang/dhall-lang/wiki/Programmable-configuration-files"class="btn btn-lg btn-outline-dark bg-light">Learn More <i class="fas fa-book"></i></a>
+      <h1 class="display-4">Don't repeat yourself</h1>
+      <hr class="my-4">
+      <p class="lead">Struggling with configuration drift?  Create a <a href="https://en.wikipedia.org/wiki/Single_source_of_truth">single source of truth</a> you can reference everywhere.</p>
+      <textarea rows="30" id="highlight0"></textarea>
+      <p>Read the same Dhall configuration file directly into several languages or convert Dhall to JSON/YAML</p>
```

Comment:
My impression was that YAML is much more important than JSON as a conversion target, but I might be wrong:

Not sure whether that's better:

```suggestion
      <p>Read the same Dhall configuration file directly into several languages or convert Dhall to YAML or other formats</p>
```

Reply:
How about "YAML, JSON, or other formats"?  I think JSON is commonly used enough that we should mention it explicitly

Reply:
Sounds good to me. :+1: